images/branding/logo/black/loopback.svg

```diff
@@ -0,0 +1 @@
+<svg id="Layer_1" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 353.17 95"><title>LoopBack-Full-Logo-(black)</title><path d="M87,72.32" transform="translate(-26.05 -26)"/><polygon points="31.11 79 43.46 71.07 59.07 79.98 74.83 70.77 87.89 78.21 59.15 95 31.11 79"/><polygon points="25.91 19.2 58.75 0 91.82 18.86 91.93 52.05 78.89 44.61 78.83 26.43 58.84 15.03 38.94 26.65 38.99 44.82 34.56 47.41 26.01 52.89 25.91 19.2"/><polygon points="43.63 31.29 53.03 36.64 43.67 42.1 43.63 31.29"/><polygon points="64.94 58.38 77.88 50.8 93.95 59.95 104.86 53.58 104.79 40.95 96.56 36.25 96.51 21.25 117.9 33.45 117.91 61.02 94.03 74.97 64.94 58.38"/><polygon points="0.14 61.63 0 33.98 21.23 21.58 21.28 36.62 13.05 41.42 13.11 54.05 24.09 60.31 74.17 31.09 74.22 46.13 24.16 75.33 0.14 61.63"/><path d="M170,87V52.51h6.51V81.2h13.59V87Z" transform="translate(-26.05 -26)"/><path d="M191.59,74c0-8.25,4.59-13.43,11.95-13.43s12,5.18,12,13.43-4.59,13.53-12,13.53S191.59,82.34,191.59,74Zm17.33,2.42V71.67c0-3.85-2.07-6-5.38-6s-5.33,2.13-5.33,6v4.79c0,3.9,2.07,6,5.33,6S208.92,80.36,208.92,76.46Z" transform="translate(-26.05 -26)"/><path d="M217.91,74c0-8.25,4.6-13.43,12-13.43s12,5.18,12,13.43-4.6,13.53-12,13.53S217.91,82.34,217.91,74Zm17.34,2.42V71.67c0-3.85-2.08-6-5.38-6s-5.34,2.13-5.34,6v4.79c0,3.9,2.08,6,5.34,6S235.25,80.36,235.25,76.46Z" transform="translate(-26.05 -26)"/><path d="M246,61.2h6.32v4.25h.2a7.12,7.12,0,0,1,7.11-4.84c6.52,0,10.12,4.84,10.12,13.43s-3.6,13.53-10.12,13.53a7.27,7.27,0,0,1-7.11-4.89h-.2V96.86H246ZM263.1,76.31V71.87c0-3.66-2.27-6.08-5.53-6.08-3,0-5.28,1.63-5.28,4.15v8.2c0,2.67,2.27,4.2,5.28,4.2C260.83,82.34,263.1,80,263.1,76.31Z" transform="translate(-26.05 -26)"/><path d="M274.51,52.51h16c5.68,0,9.13,3.5,9.13,8.84s-3,7.21-6.17,7.41v.29c3,0,7.46,2.18,7.46,8,0,5.54-3.81,9.88-8.89,9.88H274.51ZM281,66.88h8.05a3.41,3.41,0,0,0,3.76-3.66V61.69c0-2.32-1.39-3.65-3.76-3.65H281Zm0,14.57h9.09c2.47,0,3.95-1.33,3.95-3.75V76c0-2.42-1.48-3.81-3.95-3.81H281Z" transform="translate(-26.05 -26)"/><path d="M303.55,80c0-5.29,4-7.9,11-7.9H319V70.19c0-2.87-1.43-4.54-4.79-4.54a6.71,6.71,0,0,0-5.87,3.25l-3.76-3.35c1.88-3,4.89-4.94,10.08-4.94,7,0,10.67,3.31,10.67,9.18V81.94h2.56v5h-3.5c-2.72,0-4.4-1.78-4.79-4.44h-.3c-.84,3.31-3.65,5-7.51,5C306.52,87.57,303.55,84.56,303.55,80ZM319,79.28V75.87h-4C311.6,75.87,310,77,310,79v.84c0,2.07,1.43,3.11,3.9,3.11C316.69,83,319,81.7,319,79.28Z" transform="translate(-26.05 -26)"/><path d="M329.78,74c0-8.25,4.3-13.43,11.9-13.43,5.19,0,8.6,2.47,10.13,6.52l-5.19,2.32a4.81,4.81,0,0,0-4.94-3.76c-3.5,0-5.28,2.47-5.28,6.08v4.69c0,3.61,1.78,6,5.28,6,3,0,4.5-1.73,5.44-4L352,80.76c-1.68,4.54-5.29,6.81-10.28,6.81C334.08,87.57,329.78,82.34,329.78,74Z" transform="translate(-26.05 -26)"/><path d="M355.36,50.43h6.33v22h.24l3.41-4.64,5.78-6.62h7.16l-8.89,10L379.22,87h-7.51l-6.62-11.56-3.4,3.71V87h-6.33Z" transform="translate(-26.05 -26)"/></svg>
```

Comment:
there is missing new line  (and other svg files as well).

Reply:
SVG's are rendered as code on GitHub but not when served with the proper `Content-Type` header over Web. IDT we should be worrying about the new lines in `.svg` files.